cranelift-codegen/meta/src/shared/instructions.rs

```diff
@@ -2071,6 +2071,54 @@ pub(crate) fn define(
         .can_trap(true),
     );
 
+    ig.push(
```

Comment:
Forgot to tell in the previous review for average (and if you could apply it there too that'd be great), but as a way to make sure we stop regressing build-step compile time, can you put these in a different function marked as `#[inline(never)]` please? (No need to move all the existing instructions.) This for instance could go into a `simd_comparisons` function, or something appropriately named like this. Thanks!

Reply:
I went with `define_simd_arithmetic` since it seems to be in that section in the [spec](https://github.com/WebAssembly/simd/blob/master/proposals/simd/SIMD.md#lane-wise-integer-minimum).